js/base/Exchange.js

```diff
@@ -58,6 +59,7 @@ try {
     BigNumber = requireFunction ('bignumber.js') // eslint-disable-line global-require
     // we prefer bignumber.js over BN.js
     // BN        = requireFunction ('bn.js') // eslint-disable-line global-require
+    stableStringify = requireFunction ('json-stable-stringify') // eslint-disable-line global-require
```

Comment:
We want to be very picky with the deps, so, I'd rather avoid adding another one.
We'll see if we can avoid it, and, if not, then we will add it as a static dep. Can you, please, clarify the reason why would it be required?

Reply:
Understood, I could move the code for the repo into a function of the ccxt package.
https://github.com/substack/json-stable-stringify/blob/master/index.js

The reason this is being used is it requires the JSON payload to be stringified as well as order the JSON keys alphabetically. Once it is in this format it can be signed and submitted to the API endpoint.

Where would the best place to insert this code into the package?

Reply:
Removed this dependency by adding it into the JavaScript base/functions directory.